src/components/TimeCardPage/TimeSheet.js

```diff
@@ -75,15 +75,22 @@ export default function Page() {
         apiClient.addTimeEntry(timesheet); 
     }
 
-
-
     return (
         <div>
             <div  style={{"display":'flex'}}>
                 <DateSelectorCard onDateChange={updateDateRange} startDate = {startDate} endDate={endDate}/>
                 <div className="col-md-5"></div>
                 <SubmitCard/>
             </div>
+            <Tabs>
+            <TabList>
+                {userTimesheets.map(
+                    (sheet) => (
```

Comment:
Looks good - I think we may want to setup some functionality for determining what sheets to render here though - for example if they select Week X, we would only want to render the timesheets for each company that correlate to that week 

And also for companies that are missing an entry for that week, we would probably want to create a blank timesheet for them 

Since the datepicker is being refactored, you could also just create this without pulling from the datepicker and how it could work currently is just given a date -> render timesheets for that date (I.e. the companies timesheets for that startDate == date) or something like that